src/main/java/ua/cv/tim/controller/AttackArchiveController.java

```diff
@@ -0,0 +1,60 @@
+package ua.cv.tim.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import ua.cv.tim.dto.AttackArchiveDTO;
+import ua.cv.tim.dto.AttackDTO;
+import ua.cv.tim.model.AttackArchive;
+import ua.cv.tim.service.AttackArchiveService;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by rmochetc on 14.02.2017.
+ */
+@RestController
+public class AttackArchiveController {
+    @Autowired
+    private AttackArchiveService attackArchiveService;
+
+    @RequestMapping(value = "/user/attackArchive", method = RequestMethod.GET, produces = {MediaType.APPLICATION_JSON_VALUE})
```

Comment:
we use Ukrainian translation so we need to use MediaType APLLICATION_UTF8_JSON_VALUE